docs/examples/use_cases/video_superres/model/model.py

```diff
@@ -177,7 +177,7 @@ def forward(self, inputs, iteration, writer, im_out=False):
             y[:, :, fr, :, :] = self.upsample(downsampled_input)
 
             if fr == self.mi:
-                conv1_out.append(self.conv1(y[:, :, self.mi, :, :]))
+                conv1_out.append(self.conv1(y[:, :, self.mi, :, :].clone()))
```

Comment:
Newer PyTorch version complains that this is in place operation and it cannot run autograd.